odh/overlays/moc/monitoring/dashboards/operate-first/jupyterhub-usage.yaml

```diff
@@ -0,0 +1,919 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  labels:
+    app: grafana
+  name: jupyterhub-usage
+spec:
+  customFolderName: Operate-First
+  json: |
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": "-- Grafana --",
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "type": "dashboard"
+          }
+        ]
+      },
+      "editable": true,
```

Comment:
can we change this to `editable: false`? we don't want anyone to be able to change it right?

Reply:
even with editable true, the changes would eventually get overwriten by ArgoCD... I think @4n4nd lowers the confusion. :+1:

Reply:
I don't think argocd would know if the contents of a configmap were changed

Reply:
Oh.. if it doesn't update the CR, that's even more confusing... I totally agree with making this editable only via git. :slightly_smiling_face: @hemajv you can perhaps consider extracting the json to a json file and linking it via url? For easier management?

Reply:
@tumido sure I can do that. Should I place the json files in the [SRE repo](https://github.com/operate-first/SRE/tree/master/dashboards) or should I place them in the apps repo?
@4n4nd wdyt?

Reply:
yeah the SRE repo sounds good :+1:

Reply:
updated :+1: